mail/common/src/test/java/com/fsck/k9/mail/AddressTest.java

```diff
@@ -116,10 +114,7 @@ public void stringQuotationShouldCorrectlyQuote() {
 
     @Test
     public void hashCode_withoutAddress() throws Exception {
-        Address address = Address.parse("name only")[0];
-        assertNull(address.getAddress());
-        
-        address.hashCode();
+        assertEquals(0, Address.parse("name only").length);
```

Comment:
We try to use the arrange, act, assert pattern with one empty line between each block. In this case there's no "arrange" necessary. But we still want to separate "act" from "assert".

Reply:
This has been corrected.